safe-global/web-core
# fix: return `null` if no Claiming App on chain

## What it solves

Resolves #1269

## How this PR fixes it

If no Claiming App is found (it's not present on the current chain) the Governance section is not rendered.

## How to test it

Open a Safe on Goerli and observe the Governance section. Switch to Gnosis Chain and observe no Governance section.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/204581999-8dcc1ba0-6980-4fd8-bcd3-e20ced41b857.png)

![image](https://user-images.githubusercontent.com/20442784/204582061-15024ef6-0ee4-4141-8711-c7dc3f479ba7.png)
